ansible/config_sonic_basedon_testbed.yml

```diff
@@ -35,6 +35,56 @@
   tasks:
 
   - block:
+    - name: Creates telemetry directory
+      file:
+          path: /etc/sonic/telemetry
+          state: directory
+      become: true
+
+    - name: Create telemetry server private key
+      openssl_privatekey:
+          path: /etc/sonic/telemetry/streamingtelemetryserver.key
+          size: 2048
+      become: true 
+
+    - name: create telemetry server csr
+      openssl_csr:
+          path: /etc/sonic/telemetry/streamingtelemetryserver.csr
+          privatekey_path: /etc/sonic/telemetry/streamingtelemetryserver.key
+      become: true
+    
+    - name: Generate a Self Signed OpenSSL telemetry server certificate
+      openssl_certificate:
+          path: /etc/sonic/telemetry/streamingtelemetryserver.cer
+          privatekey_path: /etc/sonic/telemetry/streamingtelemetryserver.key
+          csr_path: /etc/sonic/telemetry/streamingtelemetryserver.csr
```

Comment:
What is the relationship of "create telemetry server csr" and "Generate a Self Signed OpenSSL telemetry server certificate"?  Are the .csr and .key used as the input of the latter?

Reply:
Yes. .csr and .key ar einput parameters for openssl_certificate